### DISCUSSION TITLE
内网开发怎么可以打包一个离线文档？

### DISCUSSION THREAD

**Discussion by xiaoliyuxiaotian** [antvis/X6#4507] [Q&A]
内网开发怎么可以打包一个离线文档？

**Comment by JinxMeansJinx** [comment]
![image](https://github.com/user-attachments/assets/46ef2ccc-428b-4cb1-b3de-7781141fe929)
![image](https://github.com/user-attachments/assets/c73dceb7-00ba-4b22-9f54-700320688742)
### 切换该分支，下载zip包，然后将里面的文件用nginx部署即可

**Comment by SomkeRing** [comment]
2.x版本的离线官方文档是否有提供？
